### ISSUE TITLE
nvidia-docker crash GOMP

### ISSUE THREAD

**Issue by telegraphic** [ledatelescope/bifrost#59]
Just tried running the nvidia-docker and it crashed :(

```
dancpr@bldcpr:/bldata/bifrost$ nvidia-docker run --rm -it ledatelescope/bifrost
root@ad5378c8e95f:/workspace# ls
rt@ad5378c8e95f:/# cd /bifrost/examples
root@ad5378c8e95f:/bifrost/examples# python your_first_pipeline.py
Traceback (most recent call last):
  File "your_first_pipeline.py", line 4, in <module>
    from bifrost.block import Pipeline, NumpyBlock, NumpySourceBlock
  File "build/bdist.linux-x86_64/egg/bifrost/__init__.py", line 42, in <module>
  File "build/bdist.linux-x86_64/egg/bifrost/core.py", line 29, in <module>
  File "build/bdist.linux-x86_64/egg/bifrost/libbifrost.py", line 100, in <module>
  File "build/bdist.linux-x86_64/egg/bifrost/libbifrost.py", line 97, in _load_bifrost_lib
  File "/usr/local/lib/python2.7/dist-packages/pyclibrary-0.1.4-py2.7.egg/pyclibrary/c_library.py", line 86, in __call__
    **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/pyclibrary-0.1.4-py2.7.egg/pyclibrary/c_library.py", line 180, in __init__
    self._lib_ = self._link_library(lib, convention)
  File "/usr/local/lib/python2.7/dist-packages/pyclibrary-0.1.4-py2.7.egg/pyclibrary/backends/ctypes.py", line 123, in _link_library
    return cdll.LoadLibrary(lib_path)
  File "/usr/lib/python2.7/ctypes/__init__.py", line 443, in LoadLibrary
    return self._dlltype(name)
  File "/usr/lib/python2.7/ctypes/__init__.py", line 365, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: /usr/lib/x86_64-linux-gnu/libgomp.so.1: version `GOMP_4.0' not found (required by /usr/local/lib/libbifrost.so)
```

I only resorted to this because I broke my bifrost install when updating it, and now the non-docker version complains about `hwloc_topology_load`

```
  File "/usr/lib/python2.7/ctypes/__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: /usr/local/lib/libbifrost.so: undefined symbol: hwloc_topology_load
```
(I have libopenmpi-dev installed which should give me that symbol)

A path forward either way would help!

**Comment by benbarsdell** [comment]
Sorry, this is my fault, I didn't test my latest commit properly.

I'll push a fix for it tonight. In the mean time you should be able to roll back one commit and try that out.

**Comment by benbarsdell** [comment]
Can you check whether this is fixed with https://github.com/ledatelescope/bifrost/commit/fec3562b7aa72ffdf97789bd613d42135bb87e62?

I couldn't reproduce the libgomp problem; not sure why that's happening.

**Comment by MilesCranmer** [comment]
Travis and I have not seen these issues either.

**Comment by MilesCranmer** [comment]
This is likely related to #71 !

**Comment by MilesCranmer** [comment]
Danny, are you still seeing this issue? I saw it intermittently on my build, but have not observed it since I closed #71 . It could be that my Docker did a clean build of the Bifrost image due to some fiddling I did with the layers, and an update to some dependency cleared the issue.

**Comment by telegraphic** [comment]
It seems like it has!

PS: Looks like container is missing `graphviz` python package
